ang/crmAttachment.js

```diff
@@ -156,6 +156,11 @@
         var model = $parse(attr.crmAttachments);
         scope.att = model(scope.$parent);
         scope.ts = CRM.ts(null);
+        CRM.api3('Setting', 'getvalue', {
```

Comment:
Is api3 still allowed? Api4 should work here:

```
CRM.api4('Setting', 'get', {
  select: ["maxFileSize"]
}).then(function(settings) {
  // do something with settings array
}, function(failure) {
  // handle failure
});
```

Reply:
Hmm, I grepped in the js/templates/ang directories and haven't seen the usage of api4 in it (except api4explorer code). As it is working, I've now replaced the same with the above code and haven't found any issues 👍